### PR TITLE
CODEOWNERS: create new group hubble-metrics

### PR DESCRIPTION
**Pull request by rectified95**
Adding new group for the subset of Hubble code dealing with metrics generation, handler registration etc.

### CODE REVIEW

**Review by chancez — APPROVED** [review]

**Review by michi-covalent — APPROVED** [review]

**Review by rolinh — APPROVED** [review]

File: CODEOWNERS
Comment (joestringer) [inline CODEOWNERS]: Does it make sense to also remove `@cilium/sig-hubble` from here?

Comment (rectified95) [reply CODEOWNERS]: Done.

**Comment by joestringer** [comment]
@rectified95 sure, let's get https://github.com/cilium/community/pull/172 approved and merged and I'll ensure it gets reflected into the backend. I want to make sure that the team is created and available before we merge this PR delegating control to the new group, just in case any in-flight PRs are impacted by the changes.

**Comment by joestringer** [comment]
Also looks like the docs need an update. `make -C Documentation update-codeowners` should do the trick.

File: CODEOWNERS
Comment (rolinh) [inline CODEOWNERS]: Could we be more specific here? What I would suggest is to remove `sig-hubble-api` from the codeowners for Hubble metrics and add responsibility to the hubble-metrics group to ensure changes in exposed metrics are consistent and not breaking without careful consideration.

File: CODEOWNERS
Comment (rolinh) [inline CODEOWNERS]:
```suggestion
/pkg/hubble/metrics @cilium/hubble-metrics
```

**Review by rolinh — CHANGES_REQUESTED** [review]
Actually, I think we want to remove sig-hubble-api from codeowners to the hubble metrics subsystem, see rationale below.

**Review by rolinh — CHANGES_REQUESTED** [review]
Could we please also remove the reference to Hubble metrics from the description of the responsibilities of the sig-hubble-api group given the  change in scope?

**Review by joestringer — APPROVED** [review]
The new group is created and populated. Once you've addressed Robin's feedback this should be good to merge.

**Comment by rolinh** [comment]
@aanm Sorry to disable auto-merge but my feedback has not been addressed.

**Review by rolinh — CHANGES_REQUESTED** [review]
Re-requesting a change as my feedback was dismissed
> Could we please also remove the reference to Hubble metrics from the description of the responsibilities of the sig-hubble-api group given the change in scope?

**Comment by rectified95** [comment]
@rolinh Thanks for bubbling this up for me, comment addressed .

**Review by rolinh — APPROVED** [review]
@rectified95 Thank you, lgtm!

**Pull request merged**
